starter-code/js/index.js

```diff
@@ -1,52 +1,87 @@
-function deleteItem(e){
+// function deleteItem(e){
 
-}
+// }
 
 function getPriceByProduct(itemNode){
 
-}
+  var itemQuantity = document.getElementsByTagName('input')[0]; //get user's input
```

Comment:
this is for an input tag. you should put .value at the end in order to get the value of the input. It would look like this `  var itemQuantity = document.getElementsByTagName('input')[0].value;`